quack/gemm_interface.py

```diff
@@ -895,6 +896,94 @@ def gemm_dgated_ref(
     dx = torch.stack([dgate, dup], dim=-1).reshape(PreAct.shape)
     return dx.to(out_dtype), postact.to(postact_dtype)
 
+@torch.library.custom_op(
+    "quack::gemm_symmetric_out",
+    mutates_args=("preact_out", "postact_out"),
+    device_types="cuda",
+    schema="(Tensor A, Tensor B, Tensor(a2!)? preact_out, Tensor(a3!) postact_out, Tensor? C=None, bool dynamic_scheduler=False, float alpha=1.0, float beta=1.0) -> ()",
+)
+def gemm_symmetric_out(
+    A: Tensor,  # (M, K) or (L, M, K)
+    B: Tensor,  # (K, N) or (L, K, N)
+    preact_out: Tensor,  # (M, N) or (L, M, N)
```

Comment:
there should just be "out", no "preact_out" or "postact_out". From a user perspective, it shouldn't matter how it's implemented